index.bs

```diff
@@ -12113,15 +12113,18 @@ Members and former members of the Working Group and contributors to the specific
 alphabetical order):<br>
 Adenot, Paul (Mozilla Foundation) - Specification Co-editor;
 Akhgari, Ehsan (Mozilla Foundation);
+Becker, Steven (Microsoft Corporation);
 Berkovitz, Joe (Invited Expert, affiliated with Noteflight/Hal Leonard, WG Chair from September 2013 to December 2017);
 Bossart, Pierre (Intel Corporation);
+Borins, Myles (Google, Inc)
 Buffa, Michel (NSAU;)
 Caceres, Marcos (Invited Expert);
 Cardoso, Gabriel (INRIA);
 Carlson, Eric (Apple, Inc.);
 Chen, Bin (Baidu, Inc.);
 Choi, Hongchan (Google, Inc.);
+Collichio, Lisa (Qualcomm);
 Geelnard, Marcus (Opera Software);
 Gehring, Todd (Dolby Laboratories);
-Goode, Adam (Google, Inc.); Gregan, Matthew (Mozilla Foundation); Hofmann, Bill (Dolby Laboratories); Jägenstedt, Philip (Opera Software); Kalliokoski, Jussi (Invited Expert); Kawai, Ryoya (AMEI); Lilley, Chris (W3C Staff); Lowis, Chris (Invited Expert. WG co-chair from December 2012 to September 2013, affiliated with British Broadcasting Corporation); MacDonald, Alistair (W3C Invited Experts) — WG co-chair
-from March 2011 to July 2012; Mandyam, Giridhar (Qualcomm Innovation Center, Inc); Michel, Thierry (W3C/ERCIM); Noble, Jer (Apple, Inc.); O'Callahan, Robert(Mozilla Foundation); Onumonu, Anthony (British Broadcasting Corporation); Paradis, Matthew (British Broadcasting Corporation) - WG Chair from September 2013 to present; Raman, T.V. (Google, Inc.); Rogers, Chris (Google, Inc.) – Specification Editor until August 2013; Schepers, Doug (W3C/MIT); Shires, Glen (Google, Inc.); Smith, Michael (W3C/Keio); Thereaux, Olivier (British Broadcasting Corporation); Toy, Raymond (Google, Inc.) - WG Chair from December 2017 - Present, Specification Co-editor; Verdie, Jean-Charles (MStar Semiconductor, Inc.); Wei, James (Intel Corporation); Weitnauer, Michael (IRT); Wilson, Chris (Google,Inc.) - Specification Co-editor; ZERGAOUI, Mohamed (INNOVIMAX)
+Goode, Adam (Google, Inc.); Gregan, Matthew (Mozilla Foundation); Hikawa, Kazuo (AMEI); Hofmann, Bill (Dolby Laboratories); Jägenstedt, Philip (Opera Software); Kalliokoski, Jeong, Paul Changjin (HTML5 Converged Technology Forum); Jussi (Invited Expert); Lee, WonSuk (Electronics and Telecommunications Research Institute); Kakishita, AMEI); Kawai, Ryoya (AMEI); Kostiainen, Anssi (Intel Corporation); Lilley, Chris (W3C Staff); Lowis, Chris (Invited Expert. WG co-chair from December 2012 to September 2013, affiliated with British Broadcasting Corporation); MacDonald, Alistair (W3C Invited Experts) — WG co-chair
```

Comment:
I didn't go through this list to see what changed. But since you're changing this anyway, could we put one name per line like in lines 12128 and earlier?

Oh, and Philip Jagenstedt doesn't work for Opera anymore, if that matters; he's at Google now.

Reply:
> Kalliokoski, Jeong, Paul Changjin (HTML5 Converged Technology Forum); Jussi (Invited Expert)

That looks wrong. _Kalliokoski, Jussi_ has been split in two (and I don't have a date for when they left, sorry)

Agree with @rtoy that one per line would be easier.

Reply:
Kakishita, Masahiro (AMEI)